Documentation/CodingStyle.md

```diff
@@ -496,24 +496,36 @@ UniqueObject& my_unique_object(); // Free function.
 
 ### Comments
 
-Make comments look like sentences by starting with a capital letter and ending with a period (punctuation). One exception may be end of line comments like this `if (x == y) // false for NaN`.
+Comments should be written using `//` and not `/* */`, except for the copyright notice.
+
+Make comments look like sentences by starting with a capital letter and ending with a period (punctuation).
```

Comment:
"Make comments look like" doesn't really bring home what the intention is, maybe something like this?

```
Comments should be proper sentences, start with a capital letter and end with a period.
```

Reply:
Done!